packages/dashboard/webpack.config.js

```diff
@@ -89,6 +89,18 @@ module.exports = {
                     },
                 ],
             },
+            {
+                test: /\.(png|jpg|gif|woff|woff2|eot|ttf|otf)$/,
```

Comment:
file loader is hoisted, i dont think we need file loader here

Reply:
@reza-fs  And also here. I really appreciate it.